### PR TITLE
Create end-to-end testing plan for system functions

### PR DESCRIPTION
**Pull request by realcodywburns**
This document provides a detailed end-to-end testing guide covering:
- Genesis operations (mother device init, Ledger support)
- Child disk creation (ECDSA and FROST schemes)
- Transaction signing (EVM, Bitcoin Taproot, Solana, Cosmos, Zcash)
- Signature exhaustion and reserve behavior
- Reconciliation with anomaly detection
- Nullification procedures
- Error handling and edge cases

The test plan is designed for humans to walk through all system functions using MCP and any compatible agent (Claude Desktop, VS Code, or custom clients).

### CODE REVIEW

**Comment by realcodywburns** [comment]
@copilot open a new pull request to apply changes based on the comments in [this thread](https://github.com/chippr-robotics/sigil/pull/15#pullrequestreview-3675654964)

File: docs/E2E_TEST_PLAN.md [hunk 1]
```diff
@@ -0,0 +1,1504 @@
...
+
+### Test 1.3: Transfer Agent Master Shard
+
+**Objective:** Securely transfer agent shard to agent device
+
+**Procedure:**
+
+1. On mother device, locate agent shard file:
+   ```bash
+   ls -la ~/.sigil-mother/agent_master_shard.json
```
Comment (realcodywburns) [inline docs/E2E_TEST_PLAN.md]: file currently saved on disk is as 'master_shard.json' on disk.
